### DISCUSSION TITLE
TP4 - Ejercicio 9 - Elimina Duplicados - Confusión

### DISCUSSION THREAD

**Discussion by LuchoGM** [INGCOM-UNRN-P1/cursada-2023#134] [Preguntas y respuestas] [answered]
Buenas tardes.
Que es exactamente lo que deberíamos realizar en este ejercicio eliminar duplicados o invertir la cadena?
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/105023570/58b0829c-052d-4b95-a077-331b9c8e2ca8)

**Comment by martinvilu** [comment, marked as answer]
Ver: https://github.com/INGCOM-UNRN-P1/cursada-2023/discussions/129
